src/components/NostoSession.tsx

```diff
@@ -28,7 +28,7 @@ export default function NostoSession(props?: { cart?: Cart; customer?: Customer
           .setCart(currentCart)
           .setCustomer(currentCustomer)
           .viewOther()
-          .load()
+          .load({ skipEvents: true, skipPageViews: true })
```

Comment:
Do you think `skipEvents` could be risky? This api call adds `cart` data to the API. And it's possible that we may not track `ADD_TO_CART` event